packages/e2e-tests/specs/widgets/editing-widgets.test.js

```diff
@@ -608,57 +607,34 @@ describe( 'Widgets screen', () => {
 		</div></div>",
 		}
 	` );
-		const editedWidgets = await getWidgetAreaWidgets();
-		expect( editedWidgets[ 'sidebar-1' ].length ).toBe( 2 );
-		expect( editedWidgets[ 'sidebar-1' ][ 0 ] ).toBe(
-			initialWidgets[ 'sidebar-1' ][ 0 ]
+		expect( editedSerializedWidgetAreas[ 'sidebar-1' ] ).toBe(
+			[
+				initialSerializedWidgetAreas[ 'sidebar-1' ],
+				initialSerializedWidgetAreas[ 'sidebar-1' ],
+			].join( '\n' )
 		);
 	} );
 
 	it( 'Should display legacy widgets', async () => {
-		/**
-		 * Using the classic widgets screen to simulate creating legacy widgets.
-		 */
-		await activatePlugin( 'gutenberg-test-classic-widgets' );
-		await visitAdminPage( 'widgets.php' );
-
-		const searchWidget = await find(
-			{
-				role: 'heading',
-				name: 'Search',
-				level: 3,
-			},
-			{
-				root: await page.$( '#widget-list' ),
-			}
-		);
-		await searchWidget.click();
-
-		const addWidgetButton = await find( {
-			role: 'button',
-			name: 'Add Widget',
+		// Get the default empty instance of a legacy search widget.
+		const { instance: defaultSearchInstance } = await rest( {
+			method: 'POST',
+			path: '/wp/v2/widget-types/search/encode',
+			data: { instance: {} },
 		} );
-		await addWidgetButton.click();
-
-		// Wait for the changes to be saved.
-		// TODO: Might have better ways to do this.
-		await page.waitForFunction( () => {
-			const addedSearchWidget = document.querySelector(
-				'#widgets-right .widget'
-			);
-			const spinner = addedSearchWidget.querySelector( '.spinner' );
 
-			return (
-				addedSearchWidget.classList.contains( 'open' ) &&
-				! spinner.classList.contains( 'is-active' )
-			);
+		// Create a search widget in the first sidebar using the default instance.
+		await rest( {
+			method: 'POST',
+			path: '/wp/v2/widgets',
+			data: {
+				id_base: 'search',
+				sidebar: 'sidebar-1',
+				instance: defaultSearchInstance,
+			},
```

Comment:
I feel like over time it'd be nice to see some simple helper like `await createWidgets( widgets, sidebarId )`, but this can always evolve over time with `rest` and `batch` as the foundation.